src/app/src/util/embeddedMap.js

```diff
@@ -0,0 +1,100 @@
+import sortBy from 'lodash/sortBy';
+
+import { OARFont } from './constants';
+
+const formatExistingWidth = ({ width = '1000' }) => {
+    const fullWidth = width[width.length - 1] === '%';
+    if (fullWidth) {
+        return { fullWidth, width: '100' };
+    }
+    if (!width.length) {
+        return { fullWidth, width: '1000' };
+    }
+    return { fullWidth, width };
+};
+
+export const formatExistingConfig = embedConfig => {
+    if (!embedConfig) {
+        return {
+            width: '1000',
+            color: '#3d2f8c',
+            font: OARFont,
+            includeOtherContributorFields: false,
+            height: '800',
+        };
+    }
+    return {
+        ...embedConfig,
+        ...formatExistingWidth(embedConfig),
+        color: embedConfig.color.length ? embedConfig.color : '#3d2f8c',
+        font: embedConfig.font.length ? embedConfig.font : OARFont,
```

Comment:
Why are we using `.length` on these fields rather than a basic check for a falsey value? Could we ever end up in an exceptional situation because `color` or `length` are null/undefined?

Reply:
Updated in 9d913b8 and also moved some values to constants for clarity/DRYness. I was thinking that 0s would come out as 'false' but since the numbers are strings, that isn't actually an issue.